connector/src/main/scala/com/microsoft/kusto/spark/authentication/KustoAuthentication.scala

```diff
@@ -0,0 +1,52 @@
+package com.microsoft.kusto.spark.authentication
```

Comment:
shouldn't this be implemented in our Java client?

Reply:
I am not sure. Same as above.